Packs/GoogleThreatIntelligence/Classifiers/classifier-Google_Threat_Intelligence_DTM_Alerts-_Incoming_Mapper.json

```diff
@@ -0,0 +1,357 @@
+                "GTI DTM Alert Doc Paste ID": {
+                    "complex": {
+                        "accessor": " paste_id",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Post Date": {
+                    "complex": {
+                        "accessor": " post_date",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Raw Text": {
+                    "complex": {
+                        "accessor": "raw_text",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Similar Domains": {
+                    "complex": {
+                        "accessor": "similar_domains",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Source": {
+                    "complex": {
+                        "accessor": "source",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Subject": {
+                    "complex": {
+                        "accessor": "subject",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Timestamp": {
+                    "complex": {
+                        "accessor": "timestamp",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Title": {
+                    "complex": {
+                        "accessor": "title",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Topic ID": {
+                    "complex": {
+                        "accessor": "topic_id",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Doc Type": {
+                    "complex": {
+                        "accessor": "__type",
+                        "filters": [],
+                        "root": "doc",
+                        "transformers": []
+                    }
+                },
```

Comment:
The 'GTI DTM Alert Monitor Name' and 'Tags' fields are present in `dtm_alerts_lists.json` but are missing from `dtm_alert_get.json`. Please ensure consistency in test data or clarify if these fields are conditionally present.